eng/pipelines/mgmt-auto-release.yml

```diff
@@ -27,7 +27,7 @@ extends:
               - task: Npm@1
                 inputs:
                   command: 'custom'
-                  customCommand: 'install -g @azure-tools/typespec-client-generator-cli@latest'
+                  customCommand: 'install -g @azure-tools/typespec-client-generator-cli@v0.10.0'
```

Comment:
Can you change the other installation place either?
https://github.com/Azure/azure-sdk-for-go/blob/bc736674c503a9d5bc11745c33cddda30ceda2f8/eng/scripts/automation_init.sh#L62